### ISSUE TITLE
Internal Server error 500 

### ISSUE THREAD

**Issue by gaborbalogh1** [gunthercox/ChatterBot#2015]
Hi 

I'm using your example Django app in my project with the example app.html file you 
provided and the web page loads but any time I send a message I get an error at this tag:

Django version 3.0.5, using settings 'Portfolio.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CTRL-BREAK.
[27/Jul/2020 16:55:25] "GET /homepage HTTP/1.1" 200 7810
[27/Jul/2020 16:55:25] "GET /static/Resume/css/style.css HTTP/1.1" 304 0
[27/Jul/2020 16:55:25] "GET /static/Resume/js/Animation.js HTTP/1.1" 304 0
[27/Jul/2020 16:55:25] "GET /static/Resume/js/jquery.js HTTP/1.1" 304 0
[27/Jul/2020 16:55:27] "GET /main HTTP/1.1" 200 7298
Internal Server Error: /chat
Traceback (most recent call last):
  File "C:\Users\gabor.balogh\AppData\Local\Programs\Python\Python38-32\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\gabor.balogh\AppData\Local\Programs\Python\Python38-32\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\gabor.balogh\AppData\Local\Programs\Python\Python38-32\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: __init__() takes 1 positional argument but 2 were given
[27/Jul/2020 16:55:36] "POST /chat HTTP/1.1" 500 18074
Internal Server Error: /chat
Traceback (most recent call last):
  File "C:\Users\gabor.balogh\AppData\Local\Programs\Python\Python38-32\lib\site-packages\django\core\handlers\exception.py", line 34, in inner
    response = get_response(request)
  File "C:\Users\gabor.balogh\AppData\Local\Programs\Python\Python38-32\lib\site-packages\django\core\handlers\base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\gabor.balogh\AppData\Local\Programs\Python\Python38-32\lib\site-packages\django\core\handlers\base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
TypeError: __init__() takes 1 positional argument but 2 were given
[27/Jul/2020 16:57:31] "GET /chat HTTP/1.1" 500 68140

Traceback on django console.

I have tried issue 480 which you have previously had open but the session id part did not work see the attached for the view.py file.

The cause of the problem seems to be this section in the js file called jquery.js I copied over from github

xhr.send 
[jquery_js.txt](https://github.com/gunthercox/ChatterBot/files/4983054/jquery_js.txt)

![Webpagestyling and test message](https://user-images.githubusercontent.com/25662551/88564379-0f794b80-d02b-11ea-99db-fb7a68411eb4.PNG)
![Error_Trown_ByDjango](https://user-images.githubusercontent.com/25662551/88564386-10aa7880-d02b-11ea-98b0-c788f841bf97.PNG)
[urls_py.txt](https://github.com/gunthercox/ChatterBot/files/4983030/urls_py.txt)
[views_py.txt](https://github.com/gunthercox/ChatterBot/files/4983031/views_py.txt)
[Settings_py.txt](https://github.com/gunthercox/ChatterBot/files/4983040/Settings_py.txt)

What would be the solution for the 500 error and where does it coming from is it Database or Chatterbot that is refusing the connection?

Thanks in advance,

Gabor Balogh

**Comment by gaborbalogh1** [comment]
Sorry give you the wrong issue I'd is actually this one:

https://github.com/gunthercox/ChatterBot/issues/453

Thanks,

**Comment by gaborbalogh1** [comment]
Hi this is so awesome I have actually fixed the problem and is now working as expected.
I think I had the URLs wrong because it was giving this Internal error 500 message and after further research 
I was confident that the urls are working then I have replaced the 2 view in the Views.py file and is now all good.

Also I have trained the bot after noticing that it was only returning the current time or date and had the settings in the Settings .py to use the logic adapter in some way.

Again very happy with your chatbot and looking forward to use it again in another project.

Thanks,

Gabor
